### PR TITLE
add FRZR view library

### PR DESCRIPTION
**Pull request by pakastin**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/pakastin/frzr
 * Official website (optional, not the repository): https://frzr.js.org
 * NPM package url (optional): https://www.npmjs.com/package/frzr
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 160
   - Count of watchers: 13
   - Count of forks: 12
 * NPM download stats (optional):
   - Downloads in the last day: 111
   - Downloads in the last week: 1177
   - Downloads in the last month: 7569

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@pakastin Thanks for your PR, would you please use npm auto-update instead the git auto-update as our readme says? Thanks.

File: ajax/libs/frzr/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "frzr", "tiny", "view", "library", "turboboosted", "framework", "javascript", "frontend", "client-side", "dom"
+  ],
+  "filename": "frzr.min.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/pakastin/frzr.git",
+    "basePath": "dist",
+    "files": [
+      "frzr.js",
+      "frzr.min.js"
```
Comment (PeterDaveHello) [inline ajax/libs/frzr/package.json]: You can use single `"frzr*.js"` here.

**Comment by pakastin** [comment]
Ok, hope it's alright now! ;)

**Comment by pakastin** [comment]
Thanks for adding FRZR! 👍

**Comment by PeterDaveHello** [comment]
@pakastin np, may you can mention it on readme and website :smile:

**Comment by pakastin** [comment]
It's there already:
https://github.com/pakastin/frzr#download 😉

**Comment by PeterDaveHello** [comment]
@pakastin Nice :+1:

**Pull request merged**